### PR TITLE
MPL dependencies

### PR DESCRIPTION
**Pull request by geomatician**

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -5,5 +5,17 @@ ignore:
   SNYK-DEBIAN12-POSTGRESQL15-5838227:
     - '*':
         reason: Although postgresql-15 package has been flagged as vulnerable, we're not installing postgres-15 on our base image.
-        expires: 2023-09-30T17:33:45.004Z
+        expires: 2030-01-01T00:00:00.000Z
```
Comment (jkeifer) [inline .snyk]: Is changing the expiration of this issue intentional?

File: mpl-dependencies.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+github.com/go-sql-driver/mysql
```
Comment (jkeifer) [inline mpl-dependencies.txt:3]:
```suggestion
This project has the following MPL-licensed dependencies:

github.com/go-sql-driver/mysql
```

File: .snyk [hunk 3]
```diff
@@ -5,5 +5,17 @@ ignore:
...
+  SNYK:LIC:GOLANG:GITHUB.COM:GO-SQL-DRIVER:MYSQL:MPL-2.0:
+    - '*':
+      reason: Flagged as an MPL 2.0 license issue by Snyk.
```
Comment (jkeifer) [inline .snyk]: This doesn't seem like the right reason--the reason is supposed to indicate why we have ignored the fact that snyk has flagged an issue.

Maybe this is better?

```suggestion
      reason: Dependency added to the mpl-dependencies.txt file for license compliance.
```

File: .snyk (hunk 1 above)
Comment (geomatician) [reply .snyk]: It was just to make it consistent with the others.  I can change it back to the original if that's better.

Comment (mentzerk) [reply .snyk]: Agreed - should not be changed - this is saying that we won't check again on an actual vulnerability until 2030

**Review by mentzerk — APPROVED** [review]

**Pull request merged**